.github/workflows/test.yml

```diff
@@ -18,22 +18,29 @@ jobs:
     - name: 'Test syntax check'
       run: 'rbs parse gems/**/*.rbs'
   test_on_changed_gems:
+    if: github.event_name == 'pull_request'
     runs-on: ubuntu-latest
-    container:
-      image: rubylang/ruby:3.1-focal
     steps:
     - uses: actions/checkout@v3
+    - uses: ruby/setup-ruby@v1
       with:
-        fetch-depth: 100
-    - name: 'Install dependencies'
-      run: 'bundle install --frozen'
-    - name: Get changed files
-      id: changed-files
-      uses: tj-actions/changed-files@v34.4.0
+        ruby-version: '3.1'
+        bundler-cache: true
+    - name: Get changed gems
+      id: changes
+      run: |
+        echo 'gems<<GEMS' >> $GITHUB_OUTPUT
```

Comment:
Could you use a UUID instead of `GEMS` as a delimiter? The fixed delimiter has a security risk.
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

> Warning: Make sure the delimiter you're using is randomly generated and unique for each run. For more information, see "[Understanding the risk of script injections](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections)".

I use `uuidgen` for this purpose in a private project.

```yaml
END_MARKER="$(uuidgen)"
echo "key<<${END_MARKER}
content
${END_MARKER}" >> $GITHUB_OUTPUT
```

Reply:
Thank you. I missed the point!